public/js/specberus.js

```diff
@@ -299,7 +299,7 @@ jQuery.extend({
 
     function setFormParams(options) {
         // Option "echidnaReady" processed first, as it may restrict the list of enabled profiles.
-        if (options.echidnaReady === "true") $echidnaReady.prop('checked', true);
+        if (options.echidnaReady === true) $echidnaReady.prop('checked', true);
```

Comment:
I suggest `$echidnaReady.prop('checked', options.echidnaReady);` instead.
Same in the other 2 lines below.